azkaban-db/src/main/sql/create.containerization-tables-all.sql

```diff
@@ -0,0 +1,71 @@
+CREATE TABLE IF NOT EXISTS image_types (
+  id               INT         NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  type             VARCHAR(64) NOT NULL UNIQUE,
+  description      VARCHAR(2048),
+  active           BOOLEAN,
+  deployable       VARCHAR(64),
+  created_on       TIMESTAMP       NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  created_by       VARCHAR(64)     NOT NULL,
+  modified_on      TIMESTAMP       DEFAULT NULL,
```

Comment:
It's good to have CURRENT_TIMESTAMP as default. It should not be null otherwise we won't have correct audit.

Reply:
Thanks. Added non null and default as CURRENT_TIMESTAMP